es_hypernews.py

```diff
@@ -1,5 +1,4 @@
-#!/usr/bin/env python
```

Comment:
Notice: can't find usage of this script in cms jenkins

Reply:
this was used on hypernews server via a cronjob. I think for now we can ignore this as old hypernews server is not in use any more